AutomatedTesting/Gem/PythonTests/Atom/CMakeLists.txt

```diff
@@ -173,4 +173,16 @@ if(PAL_TRAIT_BUILD_HOST_TOOLS AND PAL_TRAIT_BUILD_TESTS_SUPPORTED)
         COMPONENT
             Atom
     )
+    ly_add_pytest(
+        NAME AutomatedTesting::RGATest 
+        TEST_SUITE periodic
```

Comment:
This test should be in TEST_SUITE sandbox. `periodic` tests are ran periodically in Jenkins and this test would cause it to fails because it needs the special CMAKE variable to fetch the RGA executable.

Reply:
Thanks!